### PR TITLE
Update setup scripts to handle update of EAP to latest patch

### PR DESCRIPTION
**Pull request by hcherukuri**
Update setup scripts to handle update of EAP to latest patch. 

Adding yum update -y

### CODE REVIEW

**Comment by hcherukuri** [comment]
Kicked off Validation Actions: https://github.com/azure-javaee/rhel-jboss-templates/actions

File: eap74-rhel8-payg/src/main/scripts/jbosseap-setup-redhat.sh [hunk 1]
```diff
@@ -72,7 +75,7 @@ sudo yum install curl wget unzip vim git -y | log; flag=${PIPESTATUS[0]}
 if [[ "${JDK_VERSION,,}" == "openjdk17" ]]; then
     echo "sudo alternatives --set java java-17-openjdk.x86_64" | log; flag=${PIPESTATUS[0]}
     sudo alternatives --set java java-17-openjdk.x86_64| log; flag=${PIPESTATUS[0]}
-elif [[ "${JDK_VERSION,,}" == "openjdk11" ]]; then
+elif t[[ "${JDK_VERSION,,}" == "openjdk11" ]]; then
```
Comment (backwind1233) [inline eap74-rhel8-payg/src/main/scripts/jbosseap-setup-redhat.sh:78]: Hello @hcherukuri , I think we should remove the `t`

**Review by backwind1233 — DISMISSED** [review]

**Review by edburns — DISMISSED** [review]
Thanks for filling this gap, @hcherukuri !

File: eap74-rhel8-payg/src/main/scripts/jbosseap-setup-redhat.sh (hunk 1 above)
Comment (edburns) [reply eap74-rhel8-payg/src/main/scripts/jbosseap-setup-redhat.sh:78]: Yes, great catch @backwind1233 .

Comment (edburns) [reply eap74-rhel8-payg/src/main/scripts/jbosseap-setup-redhat.sh]: Fixed in 64a0021.

**Review by edburns — APPROVED** [review]

**Pull request merged**